python/dex/api.py

```diff
@@ -17,11 +17,22 @@ def tagged_union(name: str, members: List[type]):
   payload = type(name + "Payload", (ctypes.Union,), {"_fields_": named_members})
   union = type(name, (ctypes.Structure,), {
     "_fields_": [("tag", ctypes.c_uint64), ("payload", payload)],
-    "value": property(lambda self: getattr(self.payload, f"t{self.tag}")),
+    "value": property(
+        fget=lambda self: getattr(self.payload, f"t{self.tag}"),
+        fset=lambda self, value: setattr(self.payload, f"t{self.tag}", value)),
+    "Payload": payload,
   })
   return union
 
-CLit = tagged_union("Lit", [ctypes.c_int64, ctypes.c_int32, ctypes.c_int8, ctypes.c_double, ctypes.c_float])
+CLit = tagged_union("Lit", [
+  ctypes.c_int64,
+  ctypes.c_int32,
+  ctypes.c_uint8,
+  ctypes.c_double,
+  ctypes.c_float,
+  ctypes.c_uint32,
+  ctypes.c_uint64
```

Comment:
A matching change to this seems like it would be needed in the Julia code.
Which would be to add `UInt32` and `UInt64` to the ends of the list of types here:
https://github.com/google-research/dex-lang/blob/07550931f59bc5b4148e47ec940a177e26518872/julia/src/api_types.jl#L43

Reply:
Well it's not even a bug fix. The old code was correct, it just couldn't handle some of the cases.